client/components/pie-chart/index.js

```diff
@@ -60,7 +60,7 @@ class PieChart extends Component {
 		if ( nextProps.data !== prevState.data ) {
 			return {
 				data: nextProps.data,
-				dataTotal: sumBy( nextProps.data, ( datum ) => datum.value ),
+				dataTotal: nextProps.data.reduce( ( sum, datum ) => sum + datum.value, 0 ),
```

Comment:
`( sum, { value } ) => sum + value` is a bit nicer.

Reply:
Agreed, will update to add destructuring before merging.